### ISSUE TITLE
Gradle 7 deprecation warning

### ISSUE THREAD

**Issue by boris-petrov** [kordamp/jdeprscan-gradle-plugin#6]
Applying the `org.kordamp.gradle.jdeprscan` plugin under Gradle 7.5.1 yields the following warning:

```
Listener registration using Gradle.addBuildListener() has been deprecated. This will fail with an error in Gradle 8.0. Consult the upgrading guide for further information: https://docs.gradle.org/7.5.1/userguide/upgrading_version_7.html#task_execution_events
        at build_5umbho083f2juapuczrehnny6$_run_closure13.doCall(/home/boris/project/build.gradle:220)
        (Run with --stacktrace to get the full stack trace of this deprecation warning.)
```

I've also set `enableFeaturePreview 'STABLE_CONFIGURATION_CACHE'` in `settings.gradle`, not sure if it matters.

**Comment by aalmiray** [comment]
This might require updating the Kordamp Gradle plugin suite first, before any updates are made to jdeprscan.

**Comment by boris-petrov** [comment]
There is another deprecation warning:
```
Invocation of Task.project at execution time has been deprecated. This will fail with an error in Gradle 8.0. Consult the upgrading guide for further information: https://docs.gradle.org/7.6/userguide/upgrading_version_7.html#task_project
```
Let me know if you want me to open another issue for that.

**Comment by aalmiray** [comment]
Released in v0.11.0 -> https://github.com/kordamp/jdeprscan-gradle-plugin/releases/tag/v0.11.0
